PatteDoie/Services/Platform/PlatformService.cs

```diff
@@ -249,9 +250,13 @@ public async Task<List<PlatformHighScoreRow>> GetGameHighScores(string gameName)
             return _mapper.Map<List<PlatformHighScoreRow>>(highScores);
         }
 
-        public async Task<Boolean> IsHost(Guid playerId, Guid creatorId)
+        public async Task<Boolean> IsHost(Guid playerId, Guid creatorId, Guid lobbyId)
         {
-            var player = await GetUser(playerId);
+            var player = await GetUser(playerId, lobbyId);
+            if (player == null)
```

Comment:
tu peux simplifier en faisant un return player != null and le reste de ton return